### PR TITLE
Move iris_flowers model to onnx model zoo

### PR DESCRIPTION
**Pull request by frankfliu**
## Channel for questions ##
Before or while filing an issue please feel free to join our [Slack channel](https://join.slack.com/t/deepjavalibrary/shared_invite/zt-ar91gjkz-qbXhr1l~LFGEIEeGBibT7w) to get in touch with development team, ask questions, find out what's cooking and more!

## Description ##
(Brief description of what this PR is about)

### Essentials ###
Please feel free to remove inapplicable items for your PR.
- [ ] Changes are complete (i.e. I finished coding on this PR)
- [ ] All changes have test coverage:
- [ ] Code is well-documented: 
    - For user-facing API changes, Java doc has been updated. 
    - For new examples, README.md is added to explain the what the example does.
- [ ] To the my best knowledge, [examples](https://github.com/awslabs/djl/tree/master/examples) and [jupyter notebooks](https://github.com/awslabs/djl/tree/master/jupyter) are either not affected by this change, or have been fixed to be compatible with this change

### Changes ###
- [ ] Feature1, tests, (and when applicable, Java doc)
- [ ] Feature2, tests, (and when applicable, Java doc)

## Comments ##
- If this change is a backward incompatible change, why must this change be made?
- Interesting edge cases to note here

### CODE REVIEW

**Comment by lanking520** [comment]
Can we still keep the model outside, this one is not worthwhile to keep into our model zoo. The good part for outside is also helpful for users to get a reference on how they can load an onnx model

File: api/src/main/java/ai/djl/Application.java [hunk 1]
```diff
@@ -142,4 +142,10 @@ public int hashCode() {
         Application ANY = new Application("tabular");
         Application LINEAR_REGRESSION = new Application("tabular/linear_regression");
     }
+
+    /** The common set of applications for general machine learning. */
+    public interface GENERAL {
+        Application ANY = new Application("general");
+        Application CLASSIFICATION = new Application("general/classification");
```
Comment (zachgk) [inline api/src/main/java/ai/djl/Application.java:149]: Maybe put this under Tabular?

Comment (frankfliu) [reply api/src/main/java/ai/djl/Application.java]: done

**Comment by frankfliu** [comment]
> Can we still keep the model outside, this one is not worthwhile to keep into our model zoo. The good part for outside is also helpful for users to get a reference on how they can load an onnx model

This is the first model in onnx model, we don't have an onnx model yet. I think this is a good example to keep it in model zoo. Just like we keep mnist in our basic model zoo.

If in the example or jupyter notebook, maybe keep it as loading from url is better. But this is in our unit test code.

**Review by lanking520 — APPROVED** [review]
